src/Psalm/Internal/Scanner/DocblockParser.php

```diff
@@ -258,4 +262,27 @@ private static function resolveTags(ParsedDocblock $docblock): void
                 + ($docblock->tags['psalm-param-out'] ?? []);
         }
     }
+
+    /**
+     * @throws DocblockParseException if the @since PHP tag is malformed
+     */
+    public static function parseSincePhpVersion(ParsedDocblock $docblock): ?int
+    {
+        if (!isset($docblock->tags['since'])) {
+            return null;
+        }
+        $since = trim(reset($docblock->tags['since']));
```

Comment:
In the future this should be changed to a loop in case a non-PHP `@since` tag comes first.